sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-part-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -294,5 +452,121 @@ describe(
         DisbursementScheduleStatus.Pending,
       );
     });
+
+    /**
+     * Create and save required number of notifications for student and ministry.
+     * @param notificationsCount number of notifications to create.
+     * @param student related student.
+     * @param disbursementId related disbursement id.
+     */
+    async function saveNotifications(
+      notificationsCount: number,
+      student: Student,
+      disbursementId: number,
+      daysAhead?: number,
+    ): Promise<void> {
+      while (notificationsCount > 0 && notificationsCount--) {
+        const studentNotification = createFakeNotification(
+          {
+            user: student.user,
+            auditUser: systemUsersService.systemUser,
+            notificationMessage: {
+              id: NotificationMessageType.StudentNotificationDisbursementBlocked,
+            } as NotificationMessage,
+          },
+          {
+            initialValue: {
+              messagePayload: {
+                email_address: student.user.email,
+                template_id:
+                  NotificationMessageType.StudentNotificationDisbursementBlocked,
+                personalisation: {
+                  givenNames: student.user.firstName ?? "",
+                  lastName: student.user.lastName,
+                },
+              },
+              metadata: {
+                disbursementId,
+              },
+              createdAt: daysAhead ? addDays(daysAhead) : new Date(),
+            },
+          },
+        );
+        const ministryNotification = createFakeNotification(
+          {
+            user: systemUsersService.systemUser,
+            auditUser: systemUsersService.systemUser,
+            notificationMessage: {
+              id: NotificationMessageType.MinistryNotificationDisbursementBlocked,
+            } as NotificationMessage,
+          },
+          {
+            initialValue: {
+              messagePayload: {
+                email_address: gcNotifyService.ministryToAddress(),
+                template_id:
+                  NotificationMessageType.MinistryNotificationDisbursementBlocked,
+                personalisation: {
+                  givenNames: student.user.firstName ?? "",
+                  lastName: student.user.lastName,
+                },
+              },
+              metadata: {
+                disbursementId,
+              },
+              createdAt: daysAhead ? addDays(daysAhead) : new Date(),
+            },
+          },
+        );
+        studentNotification.dateSent = new Date();
```

Comment:
date_sent can be set inside initial values.

Reply:
This morning I was clarifying a doubt I had from @andrewsignori-aot and while discussing, I got the suggestion to keep it outside the `createNotification` because it is something specific to my test data. Hence, setting it outside.